### ISSUE TITLE
Improve a11y

### ISSUE THREAD

**Issue by equinusocio** [glidejs/glide#340]
Hi here some points that should be faced in order to improve the a11y and be wcag compliant:

- the controls buttons should have the `[disabled]` attribute if they are buttons and [`aria-disabled`](https://www.w3.org/TR/wai-aria-1.1/#aria-disabled). Also the disabled buttons should be styled accordingly.  

- The current active slide should have the [`aria-current`](https://www.w3.org/TR/wai-aria-1.1/#aria-current) attribute.
